### ISSUE TITLE
python PEP8 formatting

### ISSUE THREAD

**Issue by hitz** [StanfordBioinformatics/encode_utils#2]
For easy of outside contribution I would recommend moving to PEP8 compliance where feasible.
Specifically mandated 4 space indents.

https://www.python.org/dev/peps/pep-0008/

**Comment by hitz** [comment]
You could try this: https://pypi.python.org/pypi/autopep8 althoough it will put in extra line breaks

**Comment by nathankw** [comment]
Yep I’ll take a look, will be sometime this evening though

Nathaniel Watson
Bioinformatics Software Developer
Department of Genetics
Center for Genomics & Personalized Medicine

**Comment by nathankw** [comment]
Looks good, I should be able to make a new commit tomorrow with these updates.  Doesn't seem to harm the "#:" style comments for Sphinx, nor the Google Style comments in which I use reStructuredText.

**Comment by nathankw** [comment]
New commit 484d502 reflects PEP8 synchronization by running 
``autopep8 --aggressive --in-place --max-line-length=100``
on all Python files.  Tests ran fine, docs on RTD look good. Will make a new release in the couple of days.

**Comment by hitz** [comment]
yeah this looks great in my editor now.   There are still some stray unused imports and variables assigned but never used but that's small potatoes.

**Comment by nathankw** [comment]
Removed all unused imports.  There were these in connection.py:
datetime
pdb
time
urllib3

and pdb in utils.py.
